setup.py

```diff
@@ -7,6 +7,11 @@
 
 from setuptools import setup, find_packages
 
+# Make the current nilearn sources importable.
+__curdir__ = os.path.dirname(__file__)
+sys.path.insert(1, __curdir__)
```

Comment:
Make sure the `nilearn` we import is the one we're installing, not one in the system path.